cms/static/js/models/uploads.js

```diff
@@ -14,11 +14,11 @@ var FileUpload = Backbone.Model.extend({
     },
     validate: function(attrs, options) {
         if(attrs.selectedFile && !this.checkTypeValidity(attrs.selectedFile)) {
+            var template = _.template(
```

Comment:
Nit: I prefer this as an inline function call, rather than broken out as a variable. It makes ``template`` seem like an important concept, rather than just an implementation detail of how we are doing interpolation.